client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -158,7 +162,7 @@ export class PluginsListHeader extends PureComponent {
 				<Button
 					key="plugin-list-header__buttons-update"
 					compact
-					disabled={ ! this.props.haveUpdatesSelected }
+					disabled={ isWpcom ? ! this.props.haveUpdatesSelected : ! this.hasSelectedPlugins() }
```

Comment:
I'm curious about why we use a different condition to disable the button depending on whether it is Calypso blue or green. In other words, why do the buttons behave differently depending on the app?

Reply:
It's mostly because I wanted to preserve the same UX in Calypso Blue until we ship everything. In the near future, we will remove all `isWpcom` conditions, and they will behave exactly the same.